apps/super-app/Makefile

```diff
@@ -0,0 +1,21 @@
+.DEFAULT_GOAL:=help
+
+##@ Dev
+init:
+	npm ci
+
+dev:
+	npm start
+
+copy-common: ## Copy common files from the personal-calls common directory
+	rm -rf src/common
+	cp -r ../../personal-calls/frontend/src/common src/
+# Do some simple transformations on the files
+	find src/common -name '*.[jt]s' | xargs -I% src/common/transform.sh %
+
+##@ Deployment
+
+##@ Helpers
+
+help: ## Display this help
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m\033[0m\n"} /^[a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
```

Comment:
This command looks magical to me, it's pretty cool! Am not seeing `init` and `dev` in there though, do they require comments before they can show up in `help`?

<img width="633" alt="Screenshot 2022-12-09 at 7 19 01 PM" src="https://user-images.githubusercontent.com/41856541/206691030-5f60bedd-44ac-496e-ab5e-96331352d1e9.png">

Reply:
Yeah it's sorcery to me too. It's frankly unreadable to me, so I'm just assuming it requires comments. I'll add them.